services/http/api/lib/models/time.ts

```diff
@@ -0,0 +1,76 @@
+// import { Integer, Time } from "neo4j-driver";
+import { ModelRelatedNodesI, NeogmaInstance, NeogmaModel } from "neogma";
+import { MeasurementInstance, MeasurementModel } from "./measurement";
+
+
+export type HourProperties = {
+    hour: number
+}
+
+export type HourRelatedNodes = {
+    measurement: ModelRelatedNodesI<MeasurementModel, MeasurementInstance>
+}
+
+export type HourInstance = NeogmaInstance<HourProperties, HourRelatedNodes>;
+
+export type HourModel = NeogmaModel<HourProperties, HourRelatedNodes>;
+
+// ======================================================================================
```

Comment:
Again, I think using a comment block like this would be clearer:
```suggestion
/*
 * Timestamp
 */ 
```

Reply:
Fixed!